src/test/scala/scorex/transaction/HistoryWriterTest.scala

```diff
@@ -84,7 +87,38 @@ class HistoryWriterTest extends FunSuite with Matchers with HistoryTest {
     history.status(3) shouldBe FeatureStatus.Defined
   }
 
-  test("features rollback with block rollback") {
+  test("last block should affect feature voting the same way either liquid or saved to history") {
```

Comment:
a better test would be 
```
append
append
append
```
compared to 
```
append
append
append
append
rollback
```

this will be natural when `status` won't accept `ngState` as mentioned before